### ISSUE TITLE
Exception a must be non-empty

### ISSUE THREAD

**Issue by kl-31** [you359/Keras-FasterRCNN#3]
This exception pops up during training, but the training proceeds and the loss decreases as expected. What does the exception refer to, and is it a problem? Thanks.

**Comment by you359** [comment]
some exception can be occured when training.
but i don't know what exception occured when you train Faster R-CNN.
if you have some problem, please attach screenshot about exception.

**Comment by white2018** [comment]
> 
> 
> This exception pops up during training, but the training proceeds and the loss decreases as expected. What does the exception refer to, and is it a problem? Thanks.

@kl-31 I suppose it happens while training classifier sometimes. since it cannot be guranteed that the positive or negative samples from ROI Pooling are not empty.

This exception just stopped the classifier training in current epoch.

@you359 is it possible that the rpn loss or classifier loss is smaller than 0? Could you please paste the training screenshot? thanks

**Comment by hodaatef** [comment]
![1](https://user-images.githubusercontent.com/47797065/55849688-df661a80-5b51-11e9-8d71-45119d6de3b1.png)
